code/modules/surgery/organs/liver.dm

```diff
@@ -21,6 +21,44 @@
 	var/toxLethality = LIVER_DEFAULT_TOX_LETHALITY//affects how much damage toxins do to the liver
 	var/filterToxins = TRUE //whether to filter toxins
 
+/obj/item/organ/liver/Initialize()
+	. = ..()
+	// If the liver handles foods like a clown, it honks like a bike horn
+	// Don't think about it too much.
+	RegisterSignal(src, SIGNAL_ADDTRAIT(TRAIT_COMEDY_METABOLISM), .proc/on_comedy)
+
+/obj/item/organ/liver/proc/on_comedy()
+	// Are clown "bike" horns made from the livers of ex-clowns?
```

Comment:
Could you document this proc, maybe give it a more informative name?
Add the signal handler macro as well.